manager/src/test/java/org/apache/hertzbeat/manager/controller/AppControllerTest.java

```diff
@@ -83,6 +87,116 @@ void queryAppParamDefines() throws Exception {
                 .andReturn();
     }
 
+    @Test
+    void queryPushDefine() throws Exception {
+        // Data to make
+        Job mockJob = new Job();
+        mockJob.setId(1L);
+        mockJob.setMonitorId(1L);
+        mockJob.setCategory("os");
+        mockJob.setApp("mac");
+        mockJob.setName(new HashMap<>());
+        mockJob.setMetrics(new ArrayList<>());
+        mockJob.setConfigmap(new ArrayList<>());
+
+
+        // The interface is called to return manufactured data
+        Mockito.when(appService.getPushDefine(1L)).thenReturn(mockJob);
+
+        // Request interface
+        this.mockMvc.perform(MockMvcRequestBuilders.get("/api/apps/{monitorId}/pushdefine", 1L))
+                .andExpect(jsonPath("$.code").value((int) CommonConstants.SUCCESS_CODE))
+                .andExpect(jsonPath("$.data.id").value(1))
+                .andExpect(jsonPath("$.data.app").value("mac"))
+                .andReturn();
+    }
+    @Test
```

Comment:
There needs to be a newline between functions, otherwise checkstyle will throw an exception.